src/Pages/Chat.jsx

```diff
@@ -109,7 +65,12 @@ const ChatPage = () => {
                 <Grid item xs={3}
                 style={{height: "100%"}}
                 >
-                    <RightSide />
+                    <RightSide
+                    searchContact={() => setInputFields({ ...inputFields, searchContact: "" })}
+                    userName={user.firstName + " " + user.lastName}
```

Comment:
Here i got compile exception of cannot read property of undefined.
How can i build the page according the data and only after the server brought me the data???